src/cvv.ts

```diff
@@ -0,0 +1,55 @@
+import type { Verification } from "./types";
+
+const DEFAULT_LENGTH = 3;
+
+function includes(array: number[], thing: number): boolean {
```

Comment:
imagine my surprise to learn that `.includes` as an Array instance method didn't make it into the language until es2017. THAT'S THE YEAR I JOINED THE DAMN COMMITTEE